api/ai/new/priority_algorithm/priority/priority.py

```diff
@@ -35,7 +35,8 @@ def validate(self):
     def satisfied_by(self, students: List[Student]) -> bool:
         count = 0
         for student in students:
-            count += self.value in student.attributes.get(self.attribute_id, [])
+            if student.attributes is not None:
```

Comment:
student.attributes should never be None, if it is, something massive is wrong and we *want* it to throw an error